controllers/healthcheck_controller.go

```diff
@@ -24,7 +24,8 @@ import (
 	"github.com/ghodss/yaml"
 	"github.com/go-logr/logr"
 	"github.com/prometheus/client_golang/prometheus"
-	"k8s.io/api/core/v1"
+	cron "github.com/robfig/cron/v3"
+	v1 "k8s.io/api/core/v1"
```

Comment:
I believe that this aliasing is technically unnecessary since it will end up with the same imported name regardless BUT in this case, with a handful of other `...v1` and even `...v3` imports, all this does it make it a bit more clear to the reader exactly which import `v1` maps to, without them needing to remember how golang imports work otherwise.

no change requested

Reply:
Agree.. may be it can be apiv1 or something to make more meaningful..
for now referring as v1 as there is nothing else with v1